etc/radar-grafana/dashboards/allprojects/passive.json

```diff
@@ -179,23 +167,7 @@
       "timeFrom": null,
       "timeShift": null,
       "title": "Fitbit Wear Time Percentage",
-      "tooltip": {
-        "show": true
-      },
-      "type": "flant-statusmap-panel",
-      "useMax": true,
-      "xAxis": {
-        "labelFormat": "%a %m/%d",
-        "minBucketWidthToShowWeekends": 4,
-        "show": true,
-        "showCrosshair": true,
-        "showWeekends": true
-      },
-      "yAxis": {
-        "show": true,
-        "showCrosshair": false
-      },
-      "yAxisSort": "metrics"
+      "type": "state-timeline"
```

Comment:
Interesting. Does this also support tooltip?

Reply:
Yes, it does. It mentions that it's in beta though, it might not have been available in earlier grafana versions.